parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -316,7 +331,13 @@ static Context dataContext(Map<String, String> config) {
 
         String compressionLevel = config.getOrDefault(PARQUET_COMPRESSION_LEVEL, PARQUET_COMPRESSION_LEVEL_DEFAULT);
 
-        return new Context(rowGroupSize, pageSize, dictionaryPageSize, codec, compressionLevel);
+        int rowGroupCheckMinRecordCount = Integer.parseInt(config.getOrDefault(
+            PARQUET_ROW_GROUP_CHECK_MIN_RECORD_COUNT, PARQUET_ROW_GROUP_CHECK_MIN_RECORD_COUNT_DEFAULT));
+        int rowGroupCheckMaxRecordCount = Integer.parseInt(config.getOrDefault(
+            PARQUET_ROW_GROUP_CHECK_MAX_RECORD_COUNT, PARQUET_ROW_GROUP_CHECK_MAX_RECORD_COUNT_DEFAULT));
+
```

Comment:
I think we need to add some basic validations such as the config value is positive, max is greater than min.

Reply:
thx for pointing this out. added validations.